src/parser/commands/FunctionBlock.js

```diff
@@ -27,7 +27,7 @@ class Start extends Command {
     return this._options;
   }
 
-  get arguments() {
+  get args() {
```

Comment:
What was wrong with `arguments`? Not consistent or plainly wrong?

Reply:
lint complained about argument being a keyword elswhere....